victim-loss-identifier/src/utils/findings.ts

```diff
@@ -69,3 +70,60 @@ export function createFraudNftOrderFinding(
     ],
   });
 }
+
+export function createFpFinding(fpScammer: string, fpVictims: string[], fpData: fpTransaction[]): Finding {
+  const uniqueFpNfts = new Set<string>();
+  const uniqueFpTxHashes = new Set<string>();
+
+  fpData.forEach((fp) => {
+    fp.nfts.forEach((nft) => uniqueFpNfts.add(nft));
+    uniqueFpTxHashes.add(fp.txHash);
+  });
+
+  const labels: Label[] = [
+    Label.fromObject({
+      entity: fpScammer,
+      entityType: EntityType.Address,
+      label: "Benign",
+      confidence: 0.7,
+      remove: false,
+    }),
+    ...fpVictims.map((fpVictim) =>
+      Label.fromObject({
+        entity: fpVictim,
+        entityType: EntityType.Address,
+        label: "Victim Address",
+        confidence: 0.7,
+        remove: true,
+      })
+    ),
+    ...Array.from(uniqueFpNfts).map((nft) =>
+      Label.fromObject({
+        entity: nft,
+        entityType: EntityType.Address,
+        label: "NFT",
+        confidence: 0.7,
+        remove: true,
+      })
+    ),
+    ...Array.from(uniqueFpTxHashes).map((txHash) =>
+      Label.fromObject({
+        entity: txHash,
+        entityType: EntityType.Transaction,
+        label: "Exploit transaction",
```

Comment:
"Exploit" here?

Reply:
@RCantu92, what I actually meant was the opposite, removing "transaction" and keeping "Exploit".